test/e2e/fleet_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	e2e "agones.dev/agones/test/e2e/framework"
 	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
-	"k8s.io/api/apps/v1"
+	v1 "k8s.io/api/apps/v1"
```

Comment:
Minor thing - either we should drop the `v1` import alias (as it's the same thing), or let's make it `appsv1` (which I'm pretty sure what we call it elsewhere)

Reply:
This is VS Code thing. It was added automatically. Agree that appsv1 feets more natural.